packages/common-all/src/utils/index.ts

```diff
@@ -1186,9 +1186,27 @@ export function cleanName(name: string): string {
   return name;
 }
 
-/** Given a path on any platform, convert it to a unix style path. Avoid using this with absolute paths. */
+/**
+ * Convert Windows backslash paths to slash paths: `foo\\bar` ➔ `foo/bar`
+ * Copied from https://github.com/sindresorhus/slash with the rational that its latest version is build as an pure ESM and we currently don't support these.
+ */
+export function slash(path: string) {
+  const isExtendedLengthPath = /^\\\\\?\\/.test(path);
+
+  if (isExtendedLengthPath) {
+    return path;
+  }
+
+  return path.replace(/\\/g, "/");
+}
+
+/**
+ * Given a path on any platform, convert it to a unix style path. Avoid using this with absolute paths.
+ */
 export function normalizeUnixPath(fsPath: string): string {
-  return path.posix.normalize(fsPath.replace(/\\/g, "/"));
+  return path.posix
+    ? path.posix.normalize(fsPath.replace(/\\/g, "/")) // NOTE: `path.posix` might be not available depending on your build system. For example at the time of writing `dendron-plugin-views` does not implement a `posix` property.
+    : slash(fsPath);
```

Comment:
Now thinking about this some more we probably should not embed `slash` in here. Its not really doing a normalization of the path and therefor should be a seperate function.

Reply:
addressed here https://github.com/dendronhq/dendron/pull/3755/commits/3077f00308e59685c1834683f306b10d41328ad7